DropDown/src/DropDown.swift

```diff
@@ -879,7 +892,7 @@ extension DropDown {
 
 	/// Returns the height needed to display all cells.
 	fileprivate var tableHeight: CGFloat {
-		return tableView.rowHeight * CGFloat(dataSource.count)
+		return tableView.estimatedRowHeight * CGFloat(dataSource.count)
```

Comment:
do you really need `tableHeight` if so, as your height is dynamic it will not reflect the real height.